lib/src/store.rs

```diff
@@ -130,7 +130,12 @@ impl Store {
                 return Ok(data);
             }
         }
-        let commit = self.backend.read_commit(id).await?;
+        let commit = match self.backend.read_commit(id).await {
+            Err(BackendError::ReadAccessDenied { .. }) => {
+                unimplemented!("ReacAccessDenied is not supported for commits");
+            }
+            other => other?,
+        };
```

Comment:
will this be supported in the future? As I think having the ability to hide commits is valuable for a finer grained access model.

Reply:
Yes, that's the plan.

By the way, I think I added these checks so we (or downstream projects) will notice early if they return an unexpected error from the backend but now I don't see much reason for that. I'll remove these checks.

Reply:
> Yes, that's the plan.

Nice, I also thought a bit about the error today and I think it's to coarse for file/tree level ACLs but that's a discussion for later.  

> By the way, I think I added these checks so we (or downstream projects) will notice early if they return an unexpected error from the backend but now I don't see much reason for that. I'll remove these checks.

Makes sense, as it was essentially dead code upstream.